src/gov/nasa/worldwind/geom/coords/MGRSCoordConverter.java

```diff
@@ -275,6 +275,7 @@ else if (error_code == MGRS_NOZONE_WARNING)
             {
                 latitude = UPS.getLatitude().radians;
                 longitude = UPS.getLongitude().radians;
+                error_code = MGRS_NO_ERROR;
```

Comment:
`error_code` is at this point `MGRS_NOZONE_WARNING`, which was checked right above to direct conversion to `UPS`. So reset `error_code` to `MGRS_NO_ERROR`. That way the `error_code` at the end will be `MGRS_NO_ERROR`, which is what we want.